ext/tunemygc/tunemygc_ext.c

```diff
@@ -20,6 +21,26 @@ static VALUE sym_gc_cycle_exited;
 #endif
 
 /* From @tmm1/gctools */
+static double _tunemygc_realtime()
+{
+  struct timespec ts;
+#ifdef HAVE_CLOCK_GETTIME
+  if (clock_gettime(CLOCK_REALTIME, &ts) == -1) {
```

Comment:
Should we use MONOTONIC in tracing instead? What are the ramifications of that?

Cc @Shopify/observability